mantis-runtime/src/main/java/io/mantisrx/runtime/executor/LocalJobExecutorNetworked.java

```diff
@@ -70,8 +72,13 @@ public class LocalJobExecutorNetworked {
     private static final Logger logger = LoggerFactory.getLogger(LocalJobExecutorNetworked.class);
     private static final int numPartitions = 1;
     private static final Action0 nullAction = () -> System.exit(0);
+    private final MeterRegistry registry;
+
+
+    private LocalJobExecutorNetworked(MeterRegistry registry) {
```

Comment:
We don't need the MeterRegistry in the constructor here since this is also a leaf node. This would avoid some of the other extraneous changes.